uniffi_bindgen/src/interface/enum_.rs

```diff
@@ -94,7 +94,9 @@
 //!
 //! ```
 //! # let ci = uniffi_bindgen::interface::ComponentInterface::from_webidl(r##"
-//! # namespace example {};
+//! # namespace example {
+//! #   [Throws=Example] void fn();
```

Comment:
NIT: maybe name this `func` or something.  Using a Rust keyword confuses the issue a bit.